dirty_cat/datetime_encoder.py

```diff
@@ -0,0 +1,171 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+import numpy as np
+import pandas as pd
+from dirty_cat.utils import check_input
+from pandas.tseries.holiday import USFederalHolidayCalendar as calendar
```

Comment:
Yes I’m not sure. Maybe it’s okay as it is deactivated by default, and the documentation states that it only supports US holidays for now? Otherwise, we can maybe use https://github.com/dr-prodigy/python-holidays. Do you think a new dependency is worth it?